redis-wrapper/src/main/java/redis/RedisPipeline.java

```diff
@@ -29,6 +29,11 @@ public void hIncrement(String key, String field, long value) {
         _pipelined.hincrBy(_commonRedis.buildKeyWithPrefix(key), field, value);
     }
 
+    @Override
+    public void getMembers(String key) {
```

Comment:
Isn't this one missing the `throws Exception` specified in the interface?

Also, I think `RedisException` instead of `Exception` might be enough

Reply:
This method is not going to throw an exception.